core/src/main/codegen/templates/Parser.jj

```diff
@@ -798,6 +798,11 @@ SqlNodeList ParenthesizedQueryOrCommaListWithDefault(
         e = OrderedQueryOrExpr(firstExprContext)
     |
         e = Default()
+<#if parser.includeInsertWithOmittedValues>
+    |
+        LOOKAHEAD({ getToken(1).kind != RPAREN })
```

Comment:
Maybe you could add a comment clarifying the LOOKAHEAD statement here?

Reply:
Yeah that's a good point, I'll add one here